README.md

```diff
@@ -731,6 +731,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Jacob Dunn](https://github.com/dunnjacoba)
   - [Jacob Lepler](https://github.com/leplerjacob)
   - [Jade Kneen](https://github.com/jadekneen)
+  - [Jade Bañares](https://github.com/jade-arinal-banares)
```

Comment:
Please add your name in line 733, after Jacob Lepler.
Let us know if you need any help with that! 🙂

Reply:
Oki, thank you for the highlight 🎈